question_1.rb

```diff
@@ -0,0 +1,31 @@
+module HelperMethods 
+    class Title
+        def initialize(string) 
+            @string = string
+        end 
+
+        def titleize
+            array = @string.split(' ')
+            output = ""
+            for word in array
+                unless word == 'in' || word == 'the' || word == 'of' || word == 'and' || word   == 'or' || word == 'from'
```

Comment:
enumeration method is useful here: `unless %w(in the of and or from).include?(word)`